build/build-image/cross/Dockerfile

```diff
@@ -79,10 +79,15 @@ RUN export ETCD_VERSION=v3.0.17; \
   && ln -s ../src/etcd/etcd-${ETCD_VERSION}-linux-amd64/etcd /usr/local/bin/
 
 # TODO: Remove the need for two golang versions; when we use go1.8 for all other builds we can remove this
-ENV K8S_GOLANG_EDGE_VERSION=1.8rc2 \
+ENV K8S_GOLANG_EDGE_VERSION=1.8 \
     K8S_EDGE_GOROOT=/usr/local/go_k8s_edge
 RUN mkdir -p ${K8S_EDGE_GOROOT} \
-  && curl -sSL https://golang.org/dl/go${K8S_GOLANG_EDGE_VERSION}.linux-amd64.tar.gz | tar -xz -C ${K8S_EDGE_GOROOT} --strip-components=1
+  && curl -sSL https://golang.org/dl/go${K8S_GOLANG_EDGE_VERSION}.src.tar.gz | tar -xz -C ${K8S_EDGE_GOROOT} --strip-components=1
 
-# Prebuild the standard library for those platforms that need the edge GOROOT
-RUN for platform in linux/arm linux/ppc64le; do GOOS=${platform%/*} GOARCH=${platform##*/} GOROOT=${K8S_EDGE_GOROOT} go install std; done
+# We need a patched go1.8 for linux/ppc64le (https://github.com/golang/go/issues/19425)
+COPY golang-patches/CL38131-go1.8.patch ${K8S_EDGE_GOROOT}/
+RUN cd ${K8S_EDGE_GOROOT} \
+  && patch -p1 < CL38131-go1.8.patch \
+  && cd src \
+  && GOROOT_FINAL=${K8S_EDGE_GOROOT} GOROOT_BOOTSTRAP=/usr/local/go ./make.bash \
+  && for platform in linux/arm linux/ppc64le; do GOOS=${platform%/*} GOARCH=${platform##*/} GOROOT=${K8S_EDGE_GOROOT} go install std; done
```

Comment:
do we need to do this for `linux/arm` too? if so, update the comment on L87. If not, remove it here?